### PR TITLE
Dx 1369/sample app

### PR DESCRIPTION
**Pull request by BryanZWu**
Do NOT merge until after #22.

edit: this ended up just being code refactoring because we deemed halfway through that it was not a worthwhile feature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ telephoneNumbers:
 setup successful. To order more numbers using this setup, use "bandwidth order category <quantity>" or "bandwidth order search <quantity>"
 
 ```
-At this point, you can now use the number for [messages](#https://dev.bandwidth.com/messaging/about.html) via the API or SDKs. Should you need more (or different) numbers than the 10 that were offered, you can order more numbers using [`bandwidth order`](#order).
+At this point, you can now use the number for [messages](https://dev.bandwidth.com/messaging/about.html) via the [API](https://dev.bandwidth.com/numbers/apiReference.html) or [SDKs](https://github.com/Bandwidth/bandwidth-sdks). Should you need more (or different) numbers than the 10 that were offered, you can order more numbers using [`bandwidth order`](#order).
```
Comment (jmulford-bw) [inline README.md:82]: I'd change these links to `https://dev.bandwidth.com/messaging/methods/messages/createMessage.html` and `https://dev.bandwidth.com/sdks/about.html` for API and SDKs respectively

File: README.md [hunk 2]
```diff
@@ -198,15 +198,15 @@ siteId: 37397
 ```
 
 #### create application
-Create a voice or messaging application.
+Create a voice or messaging application. Note that messaging applications cannot have duplicate names.
```
Comment (jmulford-bw) [inline README.md:201]: Probably the only place this is ever documented 😂

File: src/utils.js [hunk 3]
```diff
@@ -147,12 +147,26 @@ const deriveOrderType = (numberAttributes) => {
...
+const sleep = (ms) => {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+/**
+ * Takes a file replaces each occurance of each pattern in its contents with the corresponding new string. 
+ * @param {string} file The path to a non-binary file.
+ * @param {[string|RegExp, string][]} patterns An array of arrays, each of which replace instances of the first element with the second. 
+ */
+const replaceFile = (file, patterns) => {
```
Comment (jmulford-bw) [inline src/utils.js:159]: A better function name may be `replacePatternInFile`. When I see `replaceFile` I think the entire file is being replaced

File: README.md (hunk 2 above)
Comment (BryanZWu) [reply README.md:201]: This is actually so sad

File: src/utils.js (hunk 3 above)
Comment (BryanZWu) [reply src/utils.js]: Done

File: README.md (hunk 1 above)
Comment (BryanZWu) [reply README.md]: Done

**Comment by BryanZWu** [comment]
Addressed the changes, gonna merge it since it's blocking something else

**Pull request merged**